README.md

```diff
@@ -1,16 +1,58 @@
+  * [rekor.0.pub](repository/repository/targets/artifact.pub): This is a dupe of `rekor.pub` and will be removed in the next root-signing event.
+  * [ctfe.pub](repository/repository/targets/artifact.pub): Certificate Transparency log key that is used for certificates issued by Fulcio.
+  * [artifact.pub](repository/repository/targets/artifact.pub): Key that signs Cosign releases.
+* [snapshot.json]((repository/repository/snapshot.json)): This snapshot the valid metadata files. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
+* [timestamp.json]((repository/repository/timestamp.json)): The timestamp refreshes the most accurate snapshot file. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
+
+
+### Root locations
+
+The current root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-tuf-root`.
+
+
+## Current Sigstore Root Keyholders 
+* Bob Callaway [June 2021 - present]:
+  - TUF Key ID `f505595165a177a41750a8e864ed1719b1edfccd5a426fd2c0ffda33ce7ff209`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/15938791).
+* Dan Lorenc [June 2021 - present]
+  - TUF Key ID `2f64fb5eac0cf94dd39bb45308b98920055e9a0d8e012a7220787834c60aef97`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/13078778).
+* Luke Hinds [June 2021 - present]
+  - TUF Key ID `bdde902f5ec668179ff5ca0dabf7657109287d690bf97e230c21d65f99155c62`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/14454335).
+* Marina Moore [June 2021 - present]
+  - TUF Key ID `eaf22372f417dd618a46f6c627dbc276e9fd30a004fc94f9be946e73f8bd090b`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/14470876).
+* Santiago Torres-Arias [June 2021 - present]
+  - TUF Key ID `f40f32044071a9365505da3d1e3be6561f6f22d0e60cf51df783999f6c3429cb`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/15938765).
 
 ### Ceremony Overview
 At the end of the ceremony, new repository metadata will be written to a `ceremony/YYYY-MM-DD/repository` directory.
 
-The ceremony will be completed in four rounds:
+The ceremony will be completed in five rounds:
 
 ![image](https://user-images.githubusercontent.com/5194569/122459506-ffd65e80-cf7e-11eb-8915-e10ac6b50594.png)
 
 * Round 1: Add Key
 * Round 2: Sign Root & Targets
-* Round 3: Sign Snapshot
-* Round 4: Sign Timestamp
+* Round 3: Sign Delegations
+* Round 4: Sign Snapshot & Timestamp
```

Comment:
nit: Can you mention rounds 1.5 and 5 in this bullet point list instead of having the extra sentence below?